test/test.js

```diff
@@ -294,8 +285,7 @@ describe('sourcemap-concat', function() {
     });
     builder = new broccoli.Builder(node);
     return builder.build().then(function(result) {
-      expectFile('nothing.js').in(result);
-      expectFile('nothing.map').in(result);
+      expectValidSourcemap('nothing.js').in(result);
```

Comment:
Although not perfect, it would seem that:

1. an empty map file is valid, if someone asks for a `map` they get a `map` even if it is empty, as long as it is well formed.
2. the validate tool seems to not handle this case correctly.
3. unless new information surfaces, we should: skip the validator for this one scenario, and continue the existing behavior